src/App.tsx

```diff
@@ -15,48 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  Default,
+  Aplhabet,
+  Length,
+}
+
+function getPreparedGoods(goods: string[], { sortField, reversed }:
+{ sortField: SortField, reversed: boolean }) {
+  let sortedGoods = [...goods];
```

Comment:
let's align your code 
```suggestion
function getPreparedGoods(
 goods: string[], 
 { sortField, reversed }: { sortField: SortField, reversed: boolean }
) {
  let sortedGoods = [...goods];
```